src/test/java/uk/gov/ons/census/casesvc/service/EventServiceTest.java

```diff
@@ -104,7 +103,7 @@ public void testProcessPrintCaseSelected() {
     // Given
     EasyRandom easyRandom = new EasyRandom();
     Case caze = easyRandom.nextObject(Case.class);
-    when(caseService.findCase(anyInt())).thenReturn(Optional.of(caze));
+    when(caseService.getCaseByCaseRef(anyInt())).thenReturn(caze);
```

Comment:
caseRef can be set here rather than 123,  otherwise the getting caze from caseRef isn't tested in the unit test

Reply:
there is adequate test coverage. Have amended anyway but probably not a useful change at all.